apps/fz_http/lib/fz_http_web/auth/json/error_handler.ex

```diff
@@ -0,0 +1,18 @@
+defmodule FzHttpWeb.Auth.JSON.ErrorHandler do
+  @moduledoc """
+  API Error Handler module implementation for Guardian.
+  """
+  use FzHttpWeb, :controller
+  require Logger
+
+  @behaviour Guardian.Plug.ErrorHandler
+
+  @impl Guardian.Plug.ErrorHandler
+  def auth_error(conn, {type, reason}, _opts) do
+    Logger.warn("JSON.ErrorHandler.auth_error. Type: #{type}. Reason: #{reason}.")
```

Comment:
We should standardize all the log messages and their JSON format. `SON.ErrorHandler.auth_error` doesn't tell anything to admins that will see it in their logs..

Reply:
Agreed, probably part of the larger Audit logging Epic. Added notes to https://github.com/firezone/firezone/issues/949

Reply:
Updated the message to be more user-friendly.